examples/ansible/setup_neuvector.yml

```
@@ -0,0 +1,86 @@
+---
+# This Ansible playbook is used to manage the NeuVector workload on a ALP host.
+# The steps are based on : https://build.opensuse.org/package/view_file/SUSE:ALP:Workloads/neuvector-demo/README.md?expand=1
+# and https://documentation.suse.com/alp/micro/html/alp-micro/available-alp-workloads.html#task-run-neuvector-with-podman
+# The playbook supports two operations: setup and removal of NeuVector.
+# The operation is determined by the 'neuvector_state' variable, which can be set to 'present' or 'absent'.
+# When 'neuvector_state' is set to 'present', the playbook will:
+#   - Set SELinux into permissive mode
+#   - Retrieve the NeuVector image
+#   - Execute nevector runlabel INSTALL
+#   - Start the NeuVector service
+#   - Enable the NeuVector service to start on a reboot
+#   - Print a message with the connection details for NeuVector
+# When 'neuvector_state' is set to 'absent', the playbook will:
+#   - Stop the NeuVector service
+#   - Execute nevector runlabel UNINSTALL
+#   - Remove the NeuVector image
+
+- name: Running the NeuVector workload
+  hosts: alphost
```

One last suggestion: As basically all the tasks run as root (at least for now) there is no point to specify `become: true` for each of them but for the whole playbook instead:
```suggestion
  hosts: alphost
  become: true
```

---

As the suggestion has been applied please also go over the individual tasks and get rid of the `become` there.

---

Changes made. Please review